apps/server/src/modules/files/entity/file-permission.entity.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+
+import { FilePermissionReferenceModel } from '../domain';
+import { FilePermissionEntity } from './file-permission.entity';
+
+describe(FilePermissionEntity.name, () => {
+	describe('constructor', () => {
+		describe('should set proper fields values', () => {
```

Comment:
you are kind of writing the tests in reverse to everywhere else.
The describe should contain the situation ("when..."), the it should contain the expected result ("[it] should...")
You are doing it the other way around.

In order to enable easier parsing and, more importantly, easier extention in the future, I would ask you to turn that around to follow the general guidelines (you can find them in more detail here https://hpi-schul-cloud.github.io/schulcloud-server/additional-documentation/nestjs-application/testing.html).

Reply:
Sure, I'll change that, thanks for the reminder - I forgot there's a guideline on the test cases organization.

Reply:
@Metauriel could you please confirm that this is the expected way of describing the tests? Just wanted to be sure we're on the same page.

Before:
```
 PASS  apps/server/src/modules/files/entity/file-security-check.entity.spec.ts
  FileSecurityCheckEntity
    constructor
      should set proper fields values
        ✓ for an empty props object (3 ms)
        ✓ from the provided complete props object (1 ms)
```

After:
```
 PASS  apps/server/src/modules/files/entity/file-security-check.entity.spec.ts
  FileSecurityCheckEntity
    constructor
      when passed an empty props object
        ✓ should return a valid FileSecurityCheckEntity object with proper default fields values (3 ms)
      when passed a complete (fully filled) props object
        ✓ should return a valid FileSecurityCheckEntity object with fields values taken from the passed props object (1 ms)

```

It's for sure more verbose, but I think it now properly reflects the pattern we want to stick to.
Please let me know what you think or maybe what else should i change in the way I implement the pattern.

Reply:
👍  these look good, Ill try to review the rest of the tests soon